con-base/src/scripts/db/concierge_database.sql

```diff
@@ -82,4 +82,64 @@ CREATE TABLE IF NOT EXISTS `concierge`.`hotel` (
     REFERENCES `concierge`.`country` (`id`)
     ON DELETE NO ACTION
     ON UPDATE NO ACTION)
-ENGINE = InnoDB;
\ No newline at end of file
+ENGINE = InnoDB;
+=======
```

Comment:
After conflict resolving, both snippets for creating table "hotel" are in script. Remove old one.